ci-operator/config/openshift/restic/openshift-restic-oadp-1.0.yaml

```diff
@@ -0,0 +1,36 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- dockerfile_path: docker/Dockerfile
+  to: restic
+releases:
+  latest:
+    release:
+      channel: fast
+      version: "4.11"
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit-test
+  optional: true
+  steps:
+    test:
+    - as: unit
+      commands: go mod vendor && go test -v ./...
```

Comment:
missing `RESTIC_TEST_FUSE=false`, should be `go mod vendor && RESTIC_TEST_FUSE=false go test -v ./...`

Reply:
Ah forgot about that change. Thanks.